### PR TITLE
Feature/ae 36 breadcrumbs

### PR DESCRIPTION
**Pull request by MattiNieminen**

### CODE REVIEW

File: src/components/Breadcrumb/breadcrumb-utils.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { breadcrumbStore } from '@/stores';
+
+export const set = value => breadcrumbStore.set(value);
```
Comment (VesaLahd) [inline src/components/Breadcrumb/breadcrumb-utils.js]: Let's keep utils as pure functions only. This causes a side effect and is basically a 1:1 callthrough to a breadcrumbStore-method invocation.

File: src/components/Yritys/ExistingYritys.svelte [hunk 2]
```diff
@@ -36,6 +37,10 @@
     },
     { text: $_('yritys.laatijat') }
   ];
+
+  $: Breadcrumb.set([[$_('yritys.yritykset'), '/#/yritykset'],
+                     [Maybe.fold('...', R.prop('nimi'), yritys), window.location.href]]);
```
Comment (VesaLahd) [inline src/components/Yritys/ExistingYritys.svelte]: Could be just me, but i think an object would be better fit here instead of array posing as tuple.

**Review by VesaLahd — APPROVED** [review]

File: src/components/Breadcrumb/breadcrumb-utils.js (hunk 1 above)
Comment (MattiNieminen) [reply src/components/Breadcrumb/breadcrumb-utils.js]: Fixed.

File: src/components/Yritys/ExistingYritys.svelte (hunk 2 above)
Comment (MattiNieminen) [reply src/components/Yritys/ExistingYritys.svelte]: Changed.

**Pull request merged**